databuilder/main.py

```diff
@@ -14,6 +15,35 @@
 
 log = structlog.getLogger()
 
+DATABUILDER_CODE = True
+
+
+def is_databuilder(tb: traceback) -> bool:
+    """
+    Check if the code in the traceback frame is
+    from the Databuilder Code Base. This aims to remove
+    traceback involving other libaries such as importlib.
+    """
+    db_globals = tb.tb_frame.f_globals
+    return "DATABUILDER_CODE" in db_globals
```

Comment:
This only identifies this specific databuilder module, but won't catch databuilder modules that the user code calls back into. I think it ought to be possible to do the filtering instead using the filename, which I think each frame in the stack trace should have.